src/app/technical-record/shared/address-form/address-form.component.ts

```diff
@@ -0,0 +1,26 @@
+import { Component, OnInit, Input } from '@angular/core';
+import { FormGroup, FormBuilder } from '@angular/forms';
+
+@Component({
+  selector: 'vtm-address-form',
+  templateUrl: './address-form.component.html'
+})
+export class AddressFormComponent {
+  @Input() parentFg: FormGroup;
```

Comment:
not the option preferred. Dont want to couple parent with child. Let pick it up later